mismi-s3/src/Mismi/S3/Commands.hs

```diff
@@ -517,13 +518,13 @@ worker source dest mode e c errs = forever $ do
                   te = throwM $ Target a out
               foldSyncMode
                 (ifM ex te cp)
-                cp
+                (copyWithMode Overwrite a out)
```

Comment:
[foldSyncMode](https://github.com/ambiata/mismi/blob/master/mismi-s3/src/Mismi/S3/Data.hs#L128-L132) expects the second argument to implement overwrite, currently `cp` would fail.